labelbox/schema/project.py

```diff
@@ -582,6 +582,25 @@ def enable_model_assisted_labeling(self, toggle: bool = True) -> bool:
         return res["project"]["showPredictionsToLabelers"][
             "showingPredictionsToLabelers"]
 
+    def bulk_import_requests(self):
+        """ passpasspass
+        """
```

Comment:
Also does this endpoint return the most recent imports first? If so lets add that to the docstring.